pom.xml

```diff
@@ -62,20 +62,20 @@
     <dependency>
       <groupId>org.jenkins-ci.modules</groupId>
       <artifactId>ssh-cli-auth</artifactId>
-      <version>1.1</version>
+      <version>1.4</version>
```

Comment:
Updated.

I've just gone with the latest release version, but 1.2 would also work. Would there be any reason to stick with a lower version?